package/MDAnalysis/coordinates/DCD.py

```diff
@@ -315,6 +315,11 @@ def timeseries(self,
             ValueError now raised instead of NoDataError for empty input
             AtomGroup
         """
+        if asel:
```

Comment:
Can you add a deprecated entry and then also allow for the new keyword please?

Reply:
Done